lib/em/version.rb

```diff
@@ -1,3 +1,3 @@
 module EventMachine
-  VERSION = "1.2.0.1"
+  VERSION = "1.2.0.2"
```

Comment:
Please don't bump the version in a PR - I'll do that when I release a group of changes.